swaylock-fancy

```diff
@@ -16,8 +16,13 @@ swaylock_cmd=(swaylock)
 declare -a outputs
 declare -a images
 
-# parse the json swaymsg for outputs
-poutputs=$(swaymsg -t get_outputs | jq -r '.[] | select(.active != false) | .name')
+# parse wm client for display names
+if [[ "$XDG_CURRENT_DESKTOP" = "Hyprland" ]]; then
```

Comment:
```suggestion
if [[ "$XDG_CURRENT_DESKTOP" == "Hyprland" ]]; then
```

Would you mind doing this? I honestly can't get it to break, but my C++ brain goes berserk seeing the single `=`